go/types/string.go

```diff
@@ -51,3 +55,12 @@ func (s String) writeTo(w nomsWriter) {
 	StringKind.writeTo(w)
 	w.writeString(string(s))
 }
+
+func (s String) valueBytes() []byte {
+	// We know the size of the buffer here so allocate it once.
+	// StringKind, Length (UVarint), UTF-8 encoded string
+	buff := make([]byte, 1+binary.MaxVarintLen64+len(s))
```

Comment:
doesn't this need to be `len([]byte(s))`?

Reply:
That is the same thing

https://play.golang.org/p/ulXSONDx1Y

Reply:
fair enough